ricky9667/HushKeyboard
# Add select input method button

## Description

This PR adds a `select input method` button and an `open main app` button.

## How to verify

1. Verify if `Open main app` button successfully opens the app.
2. Verify if `Select input method` button can open input method manager.
3. Verify if keyboard buttons works as is.

## Screenshots /  Videos

https://github.com/ricky9667/HushKeyboard/assets/55730003/a961f270-4740-4af9-aa4d-84a0c8aecfee
